yarn-project/foundation/src/fields/coordinate.ts

```diff
@@ -28,11 +28,7 @@ export class Coordinate {
     return this.fields;
   }
 
-  /**
-   * Generates a random coordinate value
-   * @returns The random coordinate
-   */
-  static random(): Coordinate {
+  static random(_a: number): Coordinate {
```

Comment:
Where is  this change from? Just a small fix you included?

Reply:
Surprised by this, I'll double check any changes outside of barretenberg